partner_email_duplicate_warn/tests/test_email_duplicate.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+
+@tagged("post_install", "-at_install")
+class TestResPartner(TransactionCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
pls use `setUpClass` to avoid running this code on _every_ test and disable tracking to speed up.
Example https://github.com/OCA/wms/blob/14.0/shopfloor/tests/test_misc.py#L12